.github/ISSUE_TEMPLATE/docs-update-issue.md

```diff
@@ -2,21 +2,37 @@
 name: Docs update issue
 about: Request an update to a doc
 title: ''
-labels: ''
+labels: 'Bug'
 assignees: saudsami, atovpeko
 
 ---
 
-**Business case**: How will this issue help our customers?
+**SDK/Product**
+
+<!-- Signaling, Broadcast Streaming, Voice... -->
+
+**Business case**:
+<!-- How will this issue help Agora customers or yourself? -->
 
 **Subject Matter Expert**:
-- *Technical*:
-- *Language*: Iain
-- *PM*: only if essential
+<!--
+- *Technical*: @saudsami
+- *Language*: @atovpeko
+-->
+
+**Doc Link**
+ <!-- Add links to any examples of the missing/incorrect docs, if applicable -->
+
+**Scope**:
+<!-- The products and documents to be updated. If you are not sure, leave it to @atovpeko. -->
+
+**Acceptance criteria**:
 
-**Scope**: The products and documents to be updated. If you are not sure, leave it to Iain.
+- technically correct
+- easy to use
+- follows the template structure
+- follows the Google Developer Style Guide
 
-**Acceptance criteria**: The documentation is technically correct, easy to use, follows the template structure and
-follows the Google Developer Style Guide.
+**Additional valuable information**
 
-**Additional valuable information**: Anything else that will help with this issue?
+<!-- Anything else that will help with this issue? (eg Screenshots) -->
```

Comment:
For example, not eg. 

(sorry, tickets for tech writers need to follow the style guide :-) )